packages/mock-account-provider/src/run_seed.ts

```diff
@@ -1,21 +1,45 @@
 import * as _ from 'lodash'
 import { CONFIG } from './parse_config'
-import type { SeedInstance, Peering } from './parse_config'
-import { createPeer } from './requesters'
+import type { SeedInstance, Account, Peering } from './parse_config'
+import { createPeer, addPeerLiquidity, createAccount } from './requesters'
+import { v4 } from 'uuid'
 
 async function setupFromSeed(config: SeedInstance): Promise<void> {
-  const peers = await Promise.all(
-    _.map(config.peers, (peer: Peering) => {
-      return createPeer(
-        config.self.graphqlUrl,
+  const peerResponses = await Promise.all(
+    _.map(config.peers, async (peer: Peering) => {
+      const peerResponse = await createPeer(
         peer.peerIlpAddress,
         peer.peerUrl,
         peer.asset,
         peer.scale
+      ).then((response) => response.peer)
+      if (!peerResponse) {
+        throw new Error('peer response not defined')
+      }
+      const transferUid = v4()
+      const liquidity = await addPeerLiquidity(
+        config.self.graphqlUrl,
+        peerResponse.id,
+        peer.initialLiquidity.toString(),
+        transferUid
+      )
+      return [peerResponse, liquidity]
+    })
+  )
+  console.log(JSON.stringify(peerResponses, null, 2))
+  const accountResponses = await Promise.all(
+    _.map(config.accounts, async (account: Account) => {
+      const createAccountResponse = await createAccount(
```

Comment:
You probably don't need the await here with the `Promise.all`. You should just be able to return `createAccount(...)`

Reply:
done